### PR TITLE
Contribution Guide and PR Template markdown - First Drafts

### PR DESCRIPTION
**Pull request by BryanF1nes**
## Because

This is the first draft of a Contribution guide.md, as well as a potential PR-template.md for future contributors.

## This PR

 - Created and Updated a Contribution Guide located within .github/ISSUE_TEMPLATE
 - Created and Updated a PR-template.md located within .github/ISSUE_TEMPLATE

## Issue

This PR does not close any related issues associated within this repo.

## Pull Request Requirements

-   [x] I have thoroughly read and understand [Deals for Devs Contribution Guide](#)
-   [x] The title of this PR follows the `location of change: brief description of change` format
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section

### CODE REVIEW

**Comment by CBID2** [comment]
Hey @BryanF1nes. It's nice to meet you! :) It would be best to raise an issue before making a pull request. Doing so does the following
1. Ensure that other proposed contributions are not be duplicated(for example,
 I created a PR template in PR #61. 😉 )   
2. Makes it easier for people and the maintainers to review your suggestion to see if it fits the project.
If you more help on how to raise an issue, check out [my blog post](https://chrissycodes.hashnode.dev/how-to-make-your-issues-requests-come-true)

**Comment by BryanF1nes** [comment]
Hi @CBID2. Nice to meet you as well! I felt it was not necessary to make a issue since this was discussed between James and I on a discord message. However, I do see that I missed your PR template request and apologize for that!

I will take a look at making the pull request so that it is pushed to the `dev` branch instead. Thanks!

**Comment by CBID2** [comment]
> Hi @CBID2. Nice to meet you as well! I felt it was not necessary to make a issue since this was discussed between James and I on a discord message. However, I do see that I missed your PR template request and apologize for that!
> 
> 
> 
> I will take a look at making the pull request so that it is pushed to the `dev` branch instead. Thanks!

Hey @BryanF1nes! :) It's all good! :) I saw `the first time contributor` label and thought you were new to open source, hence the advice I gave.

**Comment by BryanF1nes** [comment]
@CBID2 I am new to open source for sure! So any advice is always welcome! I appreciate it! Let me know if there's anything that should be done with this PR to make things smoother for everyone moving forward!

File: .github/ISSUE_TEMPLATE/contributing.md
Comment (jamesqquick) [inline .github/ISSUE_TEMPLATE/contributing.md:116]: Can we update these example branches to include prefixes like fix_details_page or feat_details_page?

File: .github/ISSUE_TEMPLATE/contributing.md
Comment (jamesqquick) [inline .github/ISSUE_TEMPLATE/contributing.md:121]: Can we add a reference here to us using Conventional Commits? https://www.conventionalcommits.org/en/v1.0.0/#summary

File: .github/ISSUE_TEMPLATE/contributing.md
Comment (jamesqquick) [inline .github/ISSUE_TEMPLATE/contributing.md:130]: We will want to specifically pull from the remote `dev` branch in most cases. Can we update the fetch line to specify the `dev` branch?

File: .github/ISSUE_TEMPLATE/contributing.md
Comment (jamesqquick) [inline .github/ISSUE_TEMPLATE/contributing.md:110]: The vast majority of branches should be created from the `dev` branch. Can we update this to reference checking out the `dev` branch and then creating the new brach?

File: .github/ISSUE_TEMPLATE/contributing.md
Comment (jamesqquick) [inline .github/ISSUE_TEMPLATE/contributing.md:146]: Can we update this branch name to include the prefix mentioned above?

File: .github/ISSUE_TEMPLATE/contributing.md
Comment (jamesqquick) [inline .github/ISSUE_TEMPLATE/contributing.md:152]: Can we add a blockquote here to call out that PRs should be made to the `dev1` branch?

File: .github/ISSUE_TEMPLATE/pr-template.md
Comment (jamesqquick) [inline .github/ISSUE_TEMPLATE/pr-template.md:1]: Can we move this file directly into the `.github` folder? I think it's a more natural fit there than in the `ISSUE_TEMPLATE` folder

**Review by jamesqquick — APPROVED** [review]

File: .github/ISSUE_TEMPLATE/contributing.md
Comment (jamesqquick) [inline .github/ISSUE_TEMPLATE/contributing.md:2]: I accidentally said "Approved" after leaving feedback lol Can you update based on the comments below?

**Review by jamesqquick — CHANGES_REQUESTED** [review]

File: .github/ISSUE_TEMPLATE/contributing.md
Comment (BryanF1nes) [reply .github/ISSUE_TEMPLATE/contributing.md:116]: Sure thing! 

I will be commenting on these as I complete them and then issue a new PR shortly.

- Updated

File: .github/ISSUE_TEMPLATE/contributing.md
Comment (BryanF1nes) [reply .github/ISSUE_TEMPLATE/contributing.md:146]: Updated example to include prefix.

- Completed

File: .github/ISSUE_TEMPLATE/contributing.md
Comment (BryanF1nes) [reply .github/ISSUE_TEMPLATE/contributing.md:152]: >  be made to the `dev1` branch?

Just to be clear this was a typo and should be `dev` branch?

File: .github/ISSUE_TEMPLATE/pr-template.md
Comment (BryanF1nes) [reply .github/ISSUE_TEMPLATE/pr-template.md:1]: File has been moved.

- Completed

Comment (CBID2) [reply .github/ISSUE_TEMPLATE/pr-template.md:1]: Hey @jamesqquick. We already have the PR template(I did it in #61), so it would be best to revert the commit where @BryanF1nes created one

Comment (BryanF1nes) [reply .github/ISSUE_TEMPLATE/pr-template.md:1]: @CBID2 just a heads up on my end I did already delete my PR template! Thanks for letting me know! :)

File: .github/ISSUE_TEMPLATE/contributing.md
Comment (BryanF1nes) [reply .github/ISSUE_TEMPLATE/contributing.md:110]: This item has been addressed.

- Completed

**Comment by BryanF1nes** [comment]
I had to delete this repo because there seemed to be some issues on my end. Plus I wanted to create a clean PR following the guidelines discussed.

**Pull request closed without merging**
